sleap/gui/app.py

```diff
@@ -59,7 +59,7 @@
 
 import sleap
 from sleap.gui.color import ColorManager
-from sleap.gui.commands import CommandContext, UpdateTopic
+from sleap.gui.commands import CommandContext, UpdateTopic, DeleteInstanceGroup
```

Comment:
We should be calling the `CommandContext.deleteInstanceGroup` method (which you will need to create).
```suggestion
from sleap.gui.commands import CommandContext, UpdateTopic
```